config/expcore/roles.lua

```diff
@@ -206,6 +207,7 @@ Roles.define_role_order{
 
 Roles.override_player_roles{
     ["Cooldude2606"]={"Senior Administrator","Moderator","Senior Backer","Supporter"},
+    ["tovernaar123"]={"Senior Administrator"},
```

Comment:
Would be best not to commit a change used for debuging.
I would imagine that at some point we will replace this list to include all the organisers.

Reply:
this is already on master but ok XD